Robot-Framework/config/variables.robot

```
@@ -23,6 +23,8 @@ Set Variables
 
     ${config}=     Read Config
     Set Global Variable  ${SERIAL_PORT}        ${config['addresses']['${DEVICE}']['serial_port']}
+    Set Global Variable  ${RELAY_SERIAL_PORT}  ${config['addresses']['relay_serial_port']}
+    Set Global Variable  ${RELAY_NUMBER}       ${config['addresses']['${DEVICE}']['relay_number']}
```

won't it fail for setup where there are no relay boards yet?

---

Oh you're right about that.. Maybe need to add those properties with empty value for now.

---

I added IP of measurement agent previously like this to variables.robot

`Run Keyword And Ignore Error  Set Global Variable  ${RPI_IP_ADDRESS}  ${config['addresses']['measurement_agent']['device_ip_address']}
`

---

I will deploy that change before merge this one.